### PR TITLE
GDB-7711 format triple and literal results

### PR DESCRIPTION
**Pull request by boyan-tonchev**
GDB-7711

## What
Added additional formatting to the "extended_table" plugin
- triple type formatting;
- literal type formatting;

## Why
There are different type of query result bindings. "triple" and "literal" was not implemented.

## How
Added implementation.

### CODE REVIEW

File: ontotext-yasgui-web-component/src/services/utils/html-builder.ts [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+  addP(content: string, classes: string): HtmlBuilder {
+    return this.append('<p')
+      .addClass(classes)
+      .append('>')
+      .append(content)
+      .append('</p>');
+  }
+
+  addCopyResourceLinkButton(uri: string, classes = ''): HtmlBuilder {
+    return this.append("<copy-resource-link-button")
```
Comment (svilenvelikov) [inline ontotext-yasgui-web-component/src/services/utils/html-builder.ts:51]: This note is not for this line but I now saw that when I click on a copy link , the page jumpt to top which is somehow annoying.

File: ontotext-yasgui-web-component/src/services/yasr/yasr-service.ts [hunk 2]
```diff
@@ -171,3 +243,80 @@ export const defaultRawResponsePluginConfiguration = {
...
+  private readonly uriToCellElementMapping: Map<string, string> = new Map<string, string>();
+  private readonly fullUriToShortUri: Map<string, string> = new Map<string, string>();
+  private prefixes: { [label: string]: string } = {};
+  private readonly shacl: boolean;
+  private readonly ontotextResourceLocation: string
+
+  constructor() {
+    this.uriToCellElementMapping = new Map<string, string>();
+    this.shacl = window.location.href.includes(YasrService.SHACL_GRAPH_URL);
+    this.ontotextResourceLocation = window.location.origin + '/resource/';
```
Comment (svilenvelikov) [inline ontotext-yasgui-web-component/src/services/yasr/yasr-service.ts:257]: This shouldn't be here

File: ontotext-yasgui-web-component/src/services/yasr/yasr-service.ts [hunk 3]
```diff
@@ -171,3 +243,80 @@ export const defaultRawResponsePluginConfiguration = {
...
+
+class CellContentContext {
+  private readonly uriToCellElementMapping: Map<string, string> = new Map<string, string>();
+  private readonly fullUriToShortUri: Map<string, string> = new Map<string, string>();
+  private prefixes: { [label: string]: string } = {};
+  private readonly shacl: boolean;
+  private readonly ontotextResourceLocation: string
+
+  constructor() {
+    this.uriToCellElementMapping = new Map<string, string>();
```
Comment (svilenvelikov) [inline ontotext-yasgui-web-component/src/services/yasr/yasr-service.ts:255]: This is already initialized inline.

**Comment by boyan-tonchev** [comment]
Jenkins retry

**Review by svilenvelikov — APPROVED** [review]

**Pull request merged**